includes/definitions/rutos-rutx.yaml

```diff
@@ -22,3 +22,4 @@ discovery:
                 - RUT2
                 - TRB500
                 - RUT956
+                - TRB25500XXXX
```

Comment:
Can you provide the sysObjectID and sysDescr for this device please for tests?

Reply:
Yes, 

Object ID
.1.3.6.1.4.1.8072.3.2.10

sysObjectID = .*.4.1.8*
sysDescr = Linux TRB2 5.4.290 #0 Fri Apr 18 10:48:02 2025 mips

Reply:
Sorry and `.1.3.6.1.4.1.48690.1.3.0`, didn't realise it was using another oid.

Reply:
Just need the output from `.1.3.6.1.4.1.48690.1.3.0` as well so we can add some test data.